src/server/commands/pass.rs

```diff
@@ -21,7 +21,7 @@ use crate::storage;
 use async_trait::async_trait;
 use futures::channel::mpsc::Sender;
 use futures::prelude::*;
-use log::warn;
+use log::{error, warn};
```

Comment:
I think this is file should be part of https://github.com/bolcom/libunftp/pull/156